src/routes/community/community.controller.ts

```diff
@@ -44,6 +45,22 @@ export class CommunityController {
     return this.communityService.getCampaignById(resourceId);
   }
 
+  @Get('/campaigns/:resourceId/activities')
+  async getCampaignActivities(
+    @Param('resourceId') resourceId: string,
+    @RouteUrlDecorator() routeUrl: URL,
+    @PaginationDataDecorator() paginationData: PaginationData,
+    @Query('holder', new ValidationPipe(AddressSchema.optional()))
```

Comment:
A file intended to validate the `holder` (`src/routes/community/entities/campaign-activity-holder.dto.entity.ts`) was also included by mistake I think.

Reply:
Removed in 4d67562054be2d64dd9b8a6648fede25cf685324.